### ISSUE TITLE
Android training set generation tool

### ISSUE THREAD

**Issue by robfitzgerald** [Wmaxlees/hackcu#2]
Given a drawing tool, send data to the server with labels so we can generate a large set of training data.  Also this will confirm our end-to-end server-client model.

Payload should be of the form:
```JSON
{
  x: int[],
  y: int[],
  shape: string
}```
